src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1429,9 +1449,15 @@ def __call__(self, *args, **kwds) -> Any:
         Any
             Return value.
         """
-        return self.service.execute_command(
+        file_purpose = self._get_file_purpose()
+        if file_purpose == "input" and bool(self.service.remote_file_handler):
+            self.service.remote_file_handler.upload(file_name=kwds["FileName"])
+        command = self.service.execute_command(
```

Comment:
We should still follow the `beforeExecute` `afterExecute` pattern even if only to keep the code tidy.

Reply:
Yes, sure.

Reply:
Done. Thank you.